packaging/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM debian:buster-20210311
+
+LABEL maintainer="k6 Developers <developers@k6.io>"
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update -y && \
+    apt-get install -y apt-utils createrepo curl git gnupg2 python3 unzip
+
+COPY ./awscli-key.gpg .
+
+ARG AWSCLI_VERSION=2.1.35
```

Comment:
Minor nitpick, but maybe this could be an environment variable in `.github/workflows/packager.yml`, instead of hardcoded in the `Dockerfile`? So we can easily bump both it and `VERSION` at the same time.

And the `Dockerfile` default behavior, if it's not specified, could be to install the latest version, which according to https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html#cliv2-linux-install is always conveniently located at `https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip` (i.e. same URL without the `-version` before the `.zip`)

Reply:
Sure, see dec4712e.

Technically this wasn't hardcoded as build args can be changed at build time, so this was just the default, but I made it fallback on the latest version if unspecified and exposed it to be set via an env var.